internal/cmd/exodus.go

```diff
@@ -133,8 +148,11 @@ func exodusMain(ctx context.Context, cfg conf.Config, args args.Config) int {
 
 	publishItems := []gw.ItemInput{}
 
+	strip := cfg.Strip()
+	logger.F("strip", strip).Info(strip)
```

Comment:
This doesn't look appropriate for an Info log since it's just dumping the value of a particular variable with no explanation.

Could it please either be Debug, or removed, or could you find a way to write a more user-friendly Info log?  If Info it should at least be a sentence or phrase and not just a value provided without context.